packages/client/modules/demo/ClientGraphQLServer.ts

```diff
@@ -243,12 +247,50 @@ class ClientGraphQLServer extends (EventEmitter as GQLDemoEmitter) {
         }
       }
     },
-    TaskFooterIntegrateMenuRootQuery: () => {
+    TaskFooterIntegrateMenuRootQuery: (_$teamId, $userId) => {
+      const user = this.db.users[0]
       return {
         viewer: {
-          ...this.db.users[0],
+          ...user,
           userOnTeam: {
-            ...this.db.users[0]
+            ...user
+          },
+          assigneeTeamMember: {
```

Comment:
-1 sorry for not seeing it earlier, but this is just the team member of the assignee, so we should search `this.db.teamMembers` for it:
```
assigneeTeamMember: this.db.teamMembers.find((teamMember) => teamMember.userId === userId)
```